iphone/Classes/TiUIiOSStepperProxy.m

```diff
@@ -18,7 +18,10 @@ -(NSString*)apiName
     return @"Ti.UI.iOS.Stepper";
 }
 
-
+-(NSNumber*)value
+{
+    return NUMINT([[(TiUIiOSStepper*)[self view]stepper] value]);
```

Comment:
The view-method is double. Using this breaks the double-precision ... Use `NUMDOUBLE` instead. Also missing space between `]stepper`.